misc/ls-extensions/modules/performance-analyzer-services/src/main/java/io/ballerina/utils/ReturnFinder.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.utils;
+
+import io.ballerina.compiler.syntax.tree.NodeVisitor;
+import io.ballerina.compiler.syntax.tree.ReturnStatementNode;
+
+/**
+ * Visitor to find the base url of endpoint.
+ *
+ * @since 2.2.3
+ */
+public class ReturnFinder extends NodeVisitor {
+
+    private boolean hasReturn;
+
+    public boolean isHasReturn() {
+
+        return hasReturn;
+    }
+
+    public void setHasReturn(boolean hasReturn) {
+
+        this.hasReturn = hasReturn;
+    }
+
+    @Override
+    public void visit(ReturnStatementNode returnStatementNode) {
+
+        hasReturn = true;
+    }
```

Comment:
Let's remove additional blank lines after the method signatures

Reply:
Fixed